app/dashboard/_components/cards/InclusionScore/InclusionScoreTimeline.tsx

```diff
@@ -0,0 +1,79 @@
+'use client'
+import React, { useState, useEffect } from 'react'
+import { getDataTeamsAndScores } from '@/utils/getDataTeamsAndScores'
+import InclusionScoreChart from '@/app/dashboard/_components/cards/InclusionScore/InclusionScoreChart'
+import CardHeader from '@/app/dashboard/_components/CardHeader'
+import { useData } from '@/hooks/useData'
+
+type AggregatedDataEntry = {
+  month: string
+  [team: string]: number | string | null
+  benchmark: number
+}
+
+const InclusionScoreTimeline = () => {
+  const [chartData, setChartData] = useState<AggregatedDataEntry[]>([])
+  const [selectedTeam, setSelectedTeam] = useState<string>('')
+  const [teamKeys, setTeamKeys] = useState<string[]>([])
+  const { data: originalData, isLoading, error } = useData('inclusionscore')
+
+  useEffect(() => {
+    if (isLoading || error || !originalData) return
+
+    const teamsToAggregate = [
+      'Finance',
+      'Marketing',
+      'HR',
+      'company_average',
+      'Sales',
+      'Engineering',
+    ]
+
+    const result = originalData?.inclusion_metrics
+    const d = getDataTeamsAndScores(result, teamsToAggregate, 'inclusion_score')
+    setChartData(d)
+  }, [isLoading, error, originalData])
+
+  useEffect(() => {
```

Comment:
i think this could be done in the other useEffect and that way we only need to run one